product_docs/docs/pgd/5/deploy-config/deploy-tpa/index.mdx

```diff
@@ -17,13 +17,13 @@ both self-hosted and in the cloud (with AWS EC2).
 
     If you want to run on your own Linux systems or VMs, you can use also use TPA to [deploy EDB Postgres Distributed directly to your own Linux hosts](/pgd/latest/quickstart/quick_start_linux) 
```

Comment:
I believe this should be a relative link.

Reply:
Yes

Reply:
Do you want me to fix these or enter a bug?